packages/client-core/src/media/PeerMedia.tsx

```diff
@@ -182,7 +182,7 @@ export const NetworkProducer = (props: { networkID: InstanceID; producerID: stri
         $to: network.hostPeerID
       })
     )
-  }, [networkState.ready.value])
+  }, [networkState.ready?.value])
```

Comment:
These ? shouldn't be necessary, the network should always exist if the reactor is mounted

Reply:
I was getting some errors about it not existing during some of the testing I was doing. Maybe it won't happen now that I've fixed the main issue?

Reply:
it can't hurt i suppose!